test/cmocka/include/mock_memory.h

```diff
@@ -6,10 +6,8 @@
  */
 
 /* Memory mock for memmap */
-#define HEAP_RUNTIME_BASE	0xBE200000
-#define HEAP_BUFFER_BASE	0xBE2F0000
-#define HEAP_SYSTEM_0_BASE	0xBE30F000
-#define HEAP_SYS_RUNTIME_0_BASE 0xBE32F000
-#define SOF_CORE_S_START	0xBE390000
-#define HEAP_RUNTIME_SHARED_BASE	0xBE3A0000
-#define HEAP_SYSTEM_SHARED_BASE		0xBE3B0000
+#define HEAP_SYSTEM_0_BASE	0xBE100000
+#define HEAP_SYS_RUNTIME_0_BASE 0xBE120000
+#define SOF_CORE_S_START	0xBE140000
+#define HEAP_RUNTIME_BASE	0xBE180000
+#define HEAP_BUFFER_BASE	0xBE1C0000
```

Comment:
not sure how anyone could review such changes without any comments on how this is inferred?